doc/template.html

```diff
@@ -67,5 +67,41 @@ <h3 class="date">$date$</h3>
 $for(include-after)$
 $include-after$
 $endfor$
+<script type="text/javascript">
+  (function() {
+    // write out a basic 'dropdown_link' reference element
+    var dropdown_link = document.createElement('a');
+    dropdown_link.setAttribute('class', 'dropdown_link');
+    dropdown_link.onclick = function(e) {
+    }
+    dropdown_link.innerHTML = ' <b>+</b>'; // the space is for padding
+    dropdown_link.style.cursor = 'pointer';
+    // hack in an ID name for the module list
+    var collapsibleList = document.querySelectorAll('nav ul li > ul > li')
+    collapsibleList.forEach(function(element) {
```

Comment:
I think I recall the return value of `document.querySelectorAll` not actually being an array in some browsers, and hence methods don't work unless you get them from the Array prototype?